File: sm5.py
def simple(a):
  for i in range(2, a):
  	if (a%i==0.0):
  		return False
  return True
  
file = open("otus.txt", "w")

x = int(input("Type a number: "))
for i in range(2, x):
	if (simple(i)):
		file.write(str(i)+" ")
		print(i, " ")
		
file.close()
			

File: sm5.cpp
#include <iostream>
#include <fstream>

using namespace std;

bool simple(int a)
{
    for(int i=2; i<=a/2; ++i)
        if(a%i==0)
        return false;
        
    return true;
}

int main() 
{
	int x; 
	cout << "Type a number: ";
	cin >> x;

	ofstream fileStream;
    fileStream.open("result.txt");
    
    cout << "Result: ";
    
	for(int i=2; i <= x; i++)
	{
		if(simple(i)) {
			cout << i << " ";	
			fileStream << i << " ";	
		}
	}
	
	fileStream.close();
}
